es/mappings.json

```diff
@@ -16,7 +16,7 @@
       },
       "osm_key": {
         "type": "string",
-        "index": "no"
+        "index":"not_analyzed"
```

Comment:
we should add `not_analyzed` to osm_value too. at the moment it is on default: `analyzed` which is not what we want http://www.elasticsearch.org/guide/en/elasticsearch/guide/current/mapping-intro.html